.github/workflows/test.yml

```diff
@@ -0,0 +1,85 @@
+---
+name: Run DTS tests
+on:
+  pull_request:
+    # branches:
+    #   - 'main'
+jobs:
+  build-dts:
+    uses: ./.github/workflows/build.yml
+    with:
+      cacheless: false
+  run-tests:
+    name: Run DTS tests
+    if: ${{ contains(join(needs.*.result, ','), 'success') }}
+    needs: build-dts
+    runs-on:
+      labels: dts-builder
+    outputs:
+      qemu_pid: ${{ steps.run_qemu.outputs.qemu_pid }}
+    steps:
+      - name: Checkout OSFV repo
+        uses: actions/checkout@v4
+        with:
+          repository: 'Dasharo/open-source-firmware-validation'
+          path: 'open-source-firmware-validation'
+          submodules: 'recursive'
+          ref: 'develop'
+      - name: Copy DTS binary
+        shell: bash
+        run: |
+          mkdir open-source-firmware-validation/scripts/ci/qemu-data
+          bmaptool copy \
+            --bmap  build/tmp/deploy/images/genericx86-64/dts-base-image-genericx86-64.wic.bmap \
+            build/tmp/deploy/images/genericx86-64/dts-base-image-genericx86-64.wic.gz \
+            open-source-firmware-validation/scripts/ci/qemu-data/dts.img
+      - name: Install requirements
+        shell: bash
+        run: |
+          cd open-source-firmware-validation
+          python3 -m virtualenv venv
+          source venv/bin/activate
+          pip install -r requirements.txt
+      - name: Run QEMU
+        shell: bash
+        id: run_qemu
+        run: |
+          cd open-source-firmware-validation/scripts/ci
+          HDD_PATH=qemu-data/dts.img ./qemu-run.sh nographic os &
+          pid=$!
+          echo "qemu_pid=$pid" >> "$GITHUB_OUTPUT"
+      - name: Create directory for logs
+        shell: bash
+        id: log_dirs
+        run: |
+          timestamp=$(date -u +%Y-%m-%dT%H:%M:%S%Z)
+          directory="/tmp/dts-test-ci-${timestamp}"
+          mkdir $directory
+          echo "directory=$directory" >> "$GITHUB_OUTPUT"
+      - name: Run tests
+        shell: bash
+        env:
+          LOG_DIR: ${{ steps.log_dirs.outputs.directory }}
+        run: |
+          cd open-source-firmware-validation
+          source venv/bin/activate
+          source ~/.secrets/dpp-keys
```

Comment:
@PLangowski Please leave a comment if this file requires prior configuration on the specific runner.
What is the required format of this file?

Reply:
https://github.com/Dasharo/meta-dts/compare/47ff65e6610439bbae300d97a61bbc682f1151d8..379f816e541920a0d8e927da36b3962d8694c493